paths/distributions/create.yaml

```diff
@@ -48,6 +48,8 @@ requestBody:
       schema:
         type: object
         title: distribution/create/parameters
+        required:
+          - name
```

Comment:
Was looking into `api/v2/distributions` [controller](https://github.com/phrase/phrase/blob/master/app/controllers/api/v2/distributions_controller.rb#L126), maybe I missed something, but it seems `project_id` if not provided, it won't set the `project`?
If thats the case, maybe `project_id` should be required as well?

Reply:
good catch! that is required as well